egs/fame/v1/local/make_fame_train_year.pl

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/perl
+#
+# Copyright 2015   David Snyder
+#           2017   Radboud University (Author: Emre Yilmaz)
+# Apache 2.0.
+# Usage: make_sre_2010_train.pl /export/corpora5/SRE/SRE2010/eval/ data/.
+
+if (@ARGV != 6) {
+  print STDERR "Usage: $0 <path-to-SRE2010-eval> <path-to-output> <task-name> <subtask-name>\n";
+  print STDERR "e.g. $0 /export/corpora5/SRE/SRE2010/eval/ data single 3sec\n";
```

Comment:
Please correct usage messages and comments referring to SRE10.

Reply:
Fixed